extensions/amp-facebook/validator-amp-facebook.protoascii

```diff
@@ -14,11 +14,27 @@
 # limitations under the license.
 #
 
-tags: {  # amp-facebook
+
+tags: {  # amp-facebook 1.0
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-facebook 1.0"
+  excludes: "amp-facebook 0.1"
+  extension_spec: {
+    name: "amp-facebook"
+    version_name: "v1.0"
+    version: "1.0"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # amp-facebook 0.1 and latest
   html_format: AMP
   tag_name: "SCRIPT"
+  satisfies: "amp-facebook 1.0"
+  excludes: "amp-facebook 0.1"
```

Comment:
Did you intend to invert these two?

If I'm reading it correctly, this is the tagspec for 0.1, but you are saying it satisfies 1.0 and excludes 0.1 usage.

Reply:
Yes, thank you.